CONTRIBUTING.rst

```diff
@@ -194,6 +194,27 @@ title                                  string
 
 For a full schema of a category JSON object, please see https://github.com/pyvideo/data/blob/master/.schemas/category.json
 
+Creating Data From Youtube Channel
+~~~~~~~~~~~~
+
+You can easily create basic data from already created youtube channel
+
+#. install requirements::
+
+   $ pip install -r tools/requirements.txt
+
+#. get youtube api key from google console, please see https://developers.google.com/youtube/registering_an_application
```

Comment:
First of all, "get youtube api key..." should be "**To** get **a** youtube api key..."  **obtain** might be a better word than **get** here. There is also no indication here that an API key is needed, so that should be stated explicitly as well. How about:
- "You'll need to obtain a YouTube API key from Google Console. For directions, see: https://developers.google.com/youtube/registering_an_application"